### DISCUSSION TITLE
You guys are the best!

### DISCUSSION THREAD

**Discussion by ghost** [Monika-After-Story/MonikaModDev#8796] [appreciation]
Omg the decorations, the snow outside, Moni's outfit! This is so amazing. Thanks so much for doing this event.

Oh, and Monika singing all I want for Christmas is you was probably the cutest thing I've ever seen. <3

**Comment by MaximumRustler** [comment]
I loved the singing part too, totally didn't expect that!

**Comment by AsuraShun** [comment]
Yeah, I loved the singing too,but the surprise of it made me laugh a bit

**Comment by MaximumRustler** [comment]
Yeah, me too. lol

**Comment by MercuryFuchsia** [comment]
To add to this, my house could have burned down Sayori style. (I wasn't Sayori in this case, however.) It's because I don't want to miss coffee with Monika and because of her encouragement to get to sleep earlier that I was able to react to it. I know it's a tragic understatement. But thanks.
